### PR TITLE
Flag Bad Candidate Facebook pages in Candidate table

### PR DESCRIPTION
**Pull request by SailingSteve**
Fix to wevote/WeVoteServer/issues/863
Consolidated some duplicate code in import_export_facebook/views_admin

### CODE REVIEW

File: candidate/controllers.py [hunk 1]
```diff
@@ -1406,14 +1410,16 @@ def save_image_to_candidate_table(candidate, image_url, source_link, kind_of_sou
...
 
     try:
-        candidate.we_vote_hosted_profile_image_url_large = we_vote_hosted_profile_image_url_large
-        candidate.we_vote_hosted_profile_image_url_medium = we_vote_hosted_profile_image_url_medium
-        candidate.we_vote_hosted_profile_image_url_tiny = we_vote_hosted_profile_image_url_tiny
+        if not url_is_broken:
+            candidate.we_vote_hosted_profile_image_url_large = we_vote_hosted_profile_image_url_large
+            candidate.we_vote_hosted_profile_image_url_medium = we_vote_hosted_profile_image_url_medium
+            candidate.we_vote_hosted_profile_image_url_tiny = we_vote_hosted_profile_image_url_tiny
         candidate.save()
```
Comment (DaleMcGrew) [inline candidate/controllers.py:1423]: Do we need to save this if not url_is_broken?

**Comment by DaleMcGrew** [comment]
@SailingSteve, I had a new idea that I didn't think of when we first spoke about this issue. When a We Vote admin tries to paste in a new Facebook page to the Candidate Edit page (on our API server's edit candidate page), could you have it check the new URL and remove the "bad_url_link" value if it is a valid Facebook URL?

**Comment by DaleMcGrew** [comment]
👍

**Comment by SailingSteve** [comment]
This was one of my test cases, I just tested again, and it seems to work.

**Pull request merged**